packages/svelte-vscode/syntaxes/svelte.tmLanguage.src.yaml

```diff
@@ -354,9 +354,12 @@ repository:
     # No quotes - just an interpolation expression.
     - include: '#interpolation'
     # Units, meaning digit characters and an optional unit string. e.g. `15px`
-    - match: ([0-9._]+[\w]{,4})(?=\s|/?>)
-      name: constant.numeric.decimal.svelte
-      patterns: [ include: '#interpolation' ]
+    - match: (?:(['"])([0-9._]+[\w%]{,4})(\1))|(?:([0-9._]+[\w%]{,4})(?=\s|/?>))
```

Comment:
The new match is a non-capturing group. Could you explain why?

Reply:
It's to more cleanly group the two alternatives in the pattern. Each of the alternatives is wrapped in a non-capturing group.